src/js_tests/StaticPaginatedSourceSpec.js

```diff
@@ -99,7 +139,7 @@
                     entries.push({id: i});
                 }
 
-                var element = new se.StaticPaginatedSource({initialElements: entries});
+                var element = new se.StaticPaginatedSource({initialElements: entries, idAttr: "id"});
```

Comment:
You should maintain this test without using the `idAttr` option, and create a new one using it. Extract common code into a shared function.